js/client-battle-tooltips.js

```diff
@@ -35,13 +35,92 @@ var BattleTooltips = (function () {
 	BattleTooltips.hideTooltip = function () {
 		$('#tooltipwrapper').html('');
 	};
-
 	// tooltips
+	// Touch delay, pressing finger more than that time will cause the tooltip to open.
+	// Shorter time will cause the button to click
+	var touchDelayForTap = 300; // ms
+
+	// Each time a finger presses the button this function will be callled
+	// First finger starts the counter, and when last fingers leaves time is checked
+	BattleTooltips._handleTouchStartFor = function (event, roomid, thing, type, elem, ownHeight) {
+		// Prevent default on touch events to block mouse events when touch is used
+		event.preventDefault();
+
+		// Saving number of Touch start happends (number of fingers on teh button,
+	  // though most of the time will be 1, don't won't a nasty bug like tooltip stuck open)
+		// Starts by 1 and increases each time this func is called
+		elem._numOfTouches = event.touches.length;
```

Comment:
A tooltip stuck open isn't really a huge issue, you can tap the tooltip to close it.

I'd prefer not to attach anything directly to an element. Something something extending native objects. Put it in the dataset if you really need it.